cardano-1.25.1.yaml

```diff
@@ -0,0 +1,157 @@
+name: cardano-1.25.1-new-stack
+
+resolver: lts-17.4
+
+packages:
+- base16-0.1.2.1
+- base16-bytestring-1.0.1.0
+- base58-bytestring-0.1.0
+- base64-0.4.2
+- bech32-1.1.0
+- bech32-th-1.0.2
+- binary-0.8.7.0
+- bimap-0.4.0
+- canonical-json-0.6.0.0
+- cborg-0.2.4.0
+- clock-0.8
+- config-ini-0.2.4.0
+- connection-0.3.1
+- containers-0.5.11.0
+- data-clist-0.1.2.2
+- dns-3.0.4
+- generic-monoid-0.1.0.0
+- generics-sop-0.5.1.0
+- ghc-byteorder-4.11.0.0.10
+- gray-code-0.3.1
+- hedgehog-1.0.2
+- hedgehog-corpus-0.2.0
+- hedgehog-quickcheck-0.1.1
+- hspec-2.7.0
```

Comment:
I wanted to use a more recent hspec for the `beforeAllWith` function when trying a thing.

I think there's no point in having this pinned to an old version.

Maybe we shouldn't drop non-test-related pinned packages though, if we want them to perfectly match what's used in cardano-node.

Reply:
Yes, we may as well use the hspec-2.7.8 version from the LTS snapshot.

If we change hedgehog versions, that may possibly break the build. We may as well leave it if it works.